client/src/components/App/Content/InvestigationForm/TabManagement/InteractionsTab/InteractionDialog/InteractionDialog.tsx

```diff
@@ -0,0 +1,78 @@
+import React from 'react';
+import { Dialog, DialogTitle, DialogContent, DialogActions, Button } from '@material-ui/core';
+import {ChevronLeft, ChevronRight} from '@material-ui/icons';
+
+import PrimaryButton from 'commons/Buttons/PrimaryButton/PrimaryButton';
+import InteractionEventDialogData from 'models/Contexts/InteractionEventDialogData';
+import {InteractionEventFormProps} from './InteractionEventForm/InteractionSection/InteractionEventForm';
+import InteractionDetailsForm from './InteractionEventForm/InteractionDetailsForm';
+import useStyles from './InteractionDialogStyles';
+
+const InteractionDialog = (props: Props) => {
+    const { isOpen, dialogTitle, loadInteractions, interactions, onDialogClose, interactionData, isNewInteraction } = props;
+    const [isAddingContacts, setIsAddingContacts] = React.useState(false);
```

Comment:
maybe use an enum that says the form states? if in the future there will be more than 2 pages...

Reply:
i think since the button is a toggle between pages and additional pages will be added inside the contact-editing form-tabs it would be redundant.... 
the enum could be added quite comfortable if needed but i dont see it happening in the near future personally.
could ask Ofek

Reply:
I'll ask her